.cirrus.yml

```diff
@@ -65,3 +66,130 @@ vagrant_task:
     else
       ssh -tt default "sudo -i make -C /vagrant localrootlessintegration"
     fi
+
+task:
+  name: ci / centos 7
+  compute_engine_instance:
+    image_project: centos-cloud
+    image: family/centos-7
+    platform: linux
+    cpu: 4
+    memory: 8
+
+  timeout_in: 30m
+  env:
+    HOME: /root
+    CIRRUS_WORKING_DIR: /home/runc
+    GO_VERSION: "1.16.4"
+    BATS_VERSION: "v1.3.0"
+
+  install_dependencies_script: |
```

Comment:
Can we now remove `Vagrantfile.centos7`?

Reply:
removed